modules/motus/downloaddb/main.nf

```diff
@@ -0,0 +1,34 @@
+process MOTUS_DOWNLOADDB {
+    label 'process_low'
+    label 'error_ignore'
+
+    conda (params.enable_conda ? "bioconda::motus=3.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/motus:3.0.1--pyhdfd78af_0' :
+        'quay.io/biocontainers/motus:3.0.1--pyhdfd78af_0' }"
+
+    input:
+    path motus_downloaddb
+
+    output:
+    path "db_mOTU"                                       , emit: db
+    path "versions.yml"                                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    """
+    ## must copy file to working directory,
+    ## otherwise the reference_db will be download to other than current directory
+    cp $motus_downloaddb dwdDB.py
+    python dwdDB.py \\
```

Comment:
I'm not sure where/what this is coming from? The mOTUs tutorial doesn't reference anything to do with this script?

It looks like the mOTUs bioconda recipe comes with the datbaase included?